tests/testthat/test-linter_tags.R

```diff
@@ -0,0 +1,20 @@
+test_that("available_linters returns a data frame", {
+  avail <- available_linters()
+  avail2 <- available_linters(c("lintr", "not-a-package"))
+  empty <- available_linters("not-a-package")
+
+  expect_is(avail, "data.frame")
```

Comment:
expect_is() is deprecated, use expect_s3_class()

Reply:
changed